mhkit/wave/io/hindcast/hindcast.py

```diff
@@ -374,7 +382,11 @@ def request_wpto_directional_spectrum(
     # Attempt to load data from cache
     hash_params = f"{lat_lon}_{year}_{tree}_{unscale}_{str_decode}_{hsds}_{path}"
     cache_dir = _get_cache_dir()
-    data, meta, _ = handle_caching(hash_params, cache_dir)
+    data, meta, _ = handle_caching(
+        hash_params,
+        cache_dir,
+        cache_content={"data": None, "metadata": None, "write_json": None},
+    )
```

Comment:
handle_caching reduced to only 5 inputs
https://pylint.readthedocs.io/en/stable/user_guide/configuration/all-options.html#max-args